dashboard/src/components/OperatorInstanceFormBody/__snapshots__/OperatorInstanceFormBody.v2.test.tsx.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`loading spinner matches the snapshot 1`] = `
+<LoadingWrapper
+  loaded={false}
+  type={0}
+/>
```

Comment:
Talking of snapshots, this one looks like the other extreme of snapshots - is there much use? (as in, a unit test can confirm this trivially without losing any clarity, imo). But fine either way.

Reply:
yep, this is just that a function that is called `itBehavesLike` generates it. This method is just syntax sugar to check if the LoadingWrapper is rendered but it also creates a snapshot.